scripts/zones/Wajaom_Woodlands/IDs.lua

```diff
@@ -39,6 +39,7 @@ zones[xi.zone.WAJAOM_WOODLANDS] =
         {
             ['Marid'] = GetTableOfIDs('Chigoe', 5),
         },
+        GHARIAL                = GetFirstID('Gharial'),
```

Comment:
sorry but where is this being used? if we're not using it, don't add it. if its being used I don't see it in the pr.

Reply:
removed it